### DISCUSSION TITLE
Is there a command to delete the plugins?

### DISCUSSION THREAD

**Discussion by FardeenCodes** [marlonrichert/zsh-snap#36] [Q&A] [answered]
I loaded plugin and now i want to delete it i know i can 
use `rm -r ~.znap/<plugin>` but is there a  command for doing this.

**Comment by marlonrichert** [comment, marked as answer]
## Update

The `znap rm` command has been retired. To delete a plugin, use `rm -r ~[<plugin name>]` or any other tool for removing files.

---

## Old Answer

Yes, there is:
```zsh
znap rm <plugin>
```

Type `znap` without arguments for a full list of all Znap commands. Type `znap help <command>` for more info on a command. Tab completion is available, too.

**Reply by FardeenCodes** [reply]
I tried this (`znap rm ohmyzsh`)
 but the output is :
`.znap.rm:zf_rm:26: /data/data/com.termux/files/home/.zsnap/ohmyzsh: is a directory
Remove  from /data/data/com.termux/files/home/.config/zsh/znap-repos? [yn] y`
may i did something wrong ?

**Reply by marlonrichert** [reply]
No, that’s a bug. I’ll fix it.

**Reply by marlonrichert** [reply]
Fixed: 36be344

**Reply by FardeenCodes** [reply]
please also remove that questions like
`zf_rm: remove "/data/data/com.termux/files/home/.zsnap/pure/screenshot-title-cmd.png'?`

**Reply by marlonrichert** [reply]
Fixed by @FardeenCodes in dedc497923b1a30e6e3b8c843ccf445e626ac5ec. Thanks!
